Library/Source/Common.h

```diff
@@ -1,13 +1,17 @@
 #pragma once
 
 #include <arcana/threading/dispatcher.h>
+#ifndef __APPLE__
 #include <filesystem>
```

Comment:
So, switching to xcode11 as min req?

Reply:
I don't have a lot of experience here. What do you think?

Reply:
I'm still in "finding unknowns" mode. Switching to xcode11 just because of std::filesystem is a bit rough. I'd rather spend some time finding the cocoa equivalent with the appropriate abstraction/if def later.

Reply:
Still not very happy with the typedef but I removed the #ifdef APPLE in runtime.cpp. So I think it's fine for a 1st iteration.

Reply:
Ok, please file an issue after merging.